src/components/editions/headline/index.tsx

```diff
@@ -171,16 +171,24 @@ const getHeadlineStyles = (
 ): SerializedStyles => {
 	const sharedStyles = getSharedStyles(format);
 
-	// Display.Immersive needs to come before Design.Interview
-	switch (format.display) {
-		case Display.Immersive:
-			return css(
-				sharedStyles,
-				getFontStyles('tight', 'bold'),
-				immersiveStyles,
-			);
-		case Display.Showcase:
-			return css(sharedStyles, getFontStyles('tight', 'bold'));
+	if (format.display === Display.Immersive) {
```

Comment:
any reason for not keeping the `switch` statements?

Reply:
We need to sit the `format.design === Design.Comment` between the two `format.display` cases

Reply:
Thanks, I thought you could do that ordering within the `switch cases`.

Also since the ordering is important it worth adding a comment so that future refactoring doesn't break this order

Reply:
yeah we can't use both `design` and `display` in the same switch statement.

added a comment 👍

Reply:
@mohammad-haque I think what Joe's saying is that we're no longer checking a single value (`display`), we're now checking both `display` _and_ `design`, which isn't possible with a `switch` (unless you use a tuple, but that's a bit awkward in JavaScript).

Reply:
Cool, thank you 👍